packages/docs/community/release-process/smart-contracts.md

```diff
@@ -59,9 +59,9 @@ For these to run, you may need to follow the [setup instructions](https://github
 
 Using these tools, a contract release candidate can be built, deployed, and proposed for upgrade automatically on a specified network. Subsequently, stakeholders can verify the release candidate against a governance upgrade proposal's contents on the network.
 
-### Verify Release on Network
+### Verify Current Bytecode on the Network
 
-Use the following script to compile contracts at a release tag and verify that the deployed network bytecode matches the compiled bytecode.
+`verify-deployed` is a script that allows you to assess whether the bytecode on the given network matches the source code of a particular commit. It will run through the Celo Core Contracts and verify that the contracts' bytecodes as specified in the `Registry` match. Here, we will want to sanity-check that our network is running the previous release's audited commit.
```

Comment:
Wdyt of changing `PREVIOUS_RELEASE` to `LATEST_RELEASE` below?

Reply:
I'd personally probably prefer `PREVIOUS_RELEASE` since `LATEST` to me could indicate release n or release n-1. But no strong opinion on my side